doc/sphinxman/source/freq.rst

```diff
@@ -50,6 +50,16 @@ the scenes, :py:func:`~psi4.frequency` is a light wrapper over
 :py:func:`~psi4.hessian` that computes the Hessian then adds a
 thermochemical analysis.
 
+Visualization of Normal Modes
+-----------------------------
+
+|PSIfour| has the ability to export a Molden file that stores information about
+the harmonic frequencies and normal modes computed using :py:func:`~psi4.frequency`.
+This feature can be enabled by setting the option |globals__normal_modes_write| to true.
+The filename of the Molden file ends in ”.molden_normal_modes, and the prefix is
+determined by |globals__writer_file_label| (if set), or else by the name of the
+output file plus the name of the current molecule.
```

Comment:
fyi, to include text from read_options keywords, you can do something like [this](https://github.com/psi4/psi4/blame/master/doc/sphinxman/source/sapt.rst#L409) which will copy it in bodily and keep the manual in sync with read_options.